README.md

```diff
@@ -1,38 +1,26 @@
-# ![nfcore/test-datasets](docs/images/test-datasets_logo.png)
-Test data to be used for automated testing with the nf-core pipelines
+# Test data set provenance
 
-> ⚠️ **Do not merge your test data to `master`! Each pipeline has a dedicated branch (and a special one for modules)**
+This test data set was generated as test data for the NBISweden/Earth-Biogenome-Project-pilot 
```

Comment:
Could you add a header before this and/or a line or two to make it more clear this is the testdata branch for genomeassembler and not NBISweden/Earth-Biogenome-Project-pilot?

Reply:
I moved this readme into the folder of the data, and updated the root readme to say it's for genomeassembler. Nice catch. Thanks.